internal/otelcollector/config/common/constants.go

```diff
@@ -105,6 +120,12 @@ const (
 	ComponentIDInsertClusterAttributesProcessor        = "resource/insert-cluster-attributes"
 	ComponentIDDropKymaAttributesProcessor             = "resource/drop-kyma-attributes"
 
+	ComponentIDSetKymaInputNameRuntimeProcessor    ComponentID = "resource/set-kyma-input-name-runtime"
+	ComponentIDSetKymaInputNameIstioProcessor      ComponentID = "resource/set-kyma-input-name-istio"
+	ComponentIDSetKymaInputNamePrometheusProcessor ComponentID = "resource/set-kyma-input-name-prometheus"
+	ComponentIDSetKymaInputNameKymaProcessor       ComponentID = "resource/set-kyma-input-name-kyma"
+	ComponentIDSetKymaInputNameK8sClusterProcessor ComponentID = "resource/set-kyma-input-name-k8s-cluster"
```

Comment:
k8scluster receiver is active when runtime input is enabled. Why do we need another a separate processor?